ykrt/src/frame/mod.rs

```diff
@@ -35,6 +36,10 @@ pub static AOT_STACKMAPS: LazyLock<Vec<SMEntry>> = LazyLock::new(|| {
     StackMapParser::get_entries(slice)
 });
 
+pub fn load_aot_stackmaps() {
```

Comment:
`pub(crate)`

Reply:
Fixed in [6ac1a43](https://github.com/ykjit/yk/pull/864/commits/6ac1a43dc7f2f7a64999aeaa54e50f88bf958052)